src/main/java/pl/edu/agh/calculationp2p/message/MessageParser.java

```diff
@@ -1,15 +1,195 @@
 package pl.edu.agh.calculationp2p.message;
 
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import pl.edu.agh.calculationp2p.message.body.*;
+import pl.edu.agh.calculationp2p.message.utils.TaskStateMess;
+import pl.edu.agh.calculationp2p.state.task.TaskState;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.util.*;
+
 public class MessageParser {
```

Comment:
MessageParser should be an interface not a class. It is used in Network module for tests